alarm/binance_utils.py

```diff
@@ -7,15 +7,32 @@
 logger = logging.getLogger(f'{__name__}')
 
 
-def get_binance_trade_pairs():
+def get_binance_exchange_info():
```

Comment:
Why do we actually need this function? We get valid trade pairs.
So why do not we name it `get_binance_valid_trade_pairs` and return a list of them?

Reply:
get_binance_exchange_info - This function needed for get response.json for all exchange info on binance. And after when we get it, in function get_binance_valid_list_of_trade_pairs we changing response in correct list and use it in model to check naming and in format_trade_pair_for_message function we use get_binance_exchange_info to get all the information about all the binary symbols, check if they match, and if they do, we take two fields baseAsset and quoteAsset from the response

Reply:
It is still a single responsibility - to take trade pairs info from Binance. Just return a dict of dicts,  like: {'LUNAUSDT': {baseAsset: 'LUNA', quoteAsset: 'USDT'}, ...}

Reply:
++